Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -77,6 +77,12 @@ public sealed partial class HumanoidCharacterProfile : ICharacterProfile
         [DataField]
         public ProtoId<SpeciesPrototype> Species { get; set; } = SharedHumanoidAppearanceSystem.DefaultSpecies;
 
+        /// <summary>
+        /// Emberfall: Custom species name.
+        /// </summary>
+        [DataField]
+        public string CustomSpeciesName { get; set; } = string.Empty;
+
```

Comment:
If you intend to make a ton of additions to `HumanoidCharacterProfile` I would recommend creating a separate class for it (or even possibly extending `HumanoidCharacterProfile`)

Reply:
probably not going to be an issue in the near future so i'll cross that bridge when i get to it